code/modules/hivemind/machines.dm

```diff
@@ -1,6 +1,7 @@
 //Hivemind various machines
 
-#define HIVE_FACTION "hive"
+#define HIVE_FACTION 			"hive"
+#define REGENERATION_SPEED 		4
```

Comment:
After extensive testing with the blobs, i'm going to say that the core needs to have no regen.

In this case, the nodes, the source object, whatever. The big thing where all the wires come from. Make sure it has no regen, if players make a suicidal dive to damage it, any hits they land should have a permanant effect

Every other machine having regen is fine, and give the cores a larger health pool to compensate if you like